contributing/new-page-template.md

```diff
@@ -0,0 +1,55 @@
+---
+# This section is called the "frontmatter" for your page
+title: "Title For Your Page" # Use title case for titles
+#linkTitle: "Link For This Page in the Sidebar"
+# The linkTitle field (above) is optional; use it to provide a shorter link if your page title is very long
+weight: 01 # This affects the placement of the link in the sidebar on the left. Pages are ordered from top to bottom by weight.
```

Comment:
Should start and go by 10s to allow insertion of future topics. Maybe also clarify that these are relative to the weight of other files?